source/tyr/migrations/versions/3e56c7e0a4a4_create_billing_plan_table.py

```diff
@@ -0,0 +1,43 @@
+"""Create billing_plan table
+
+Revision ID: 3e56c7e0a4a4
+Revises: 3aaddd5707bd
+Create Date: 2015-11-05 13:30:32.460413
+
+"""
+
+revision = '3e56c7e0a4a4'
+down_revision = '3aaddd5707bd'
+
+from alembic import op, context
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.types import Enum
+
+def upgrade():
+
+    op.create_table('billing_plan',
+        sa.Column(
+            'id', sa.Integer(),
+            primary_key=True,
+            nullable=False
+        ),
+        sa.Column('name', sa.String(255), nullable=False),
+        sa.Column('max_request_count', sa.Integer(), nullable=True),
+        sa.Column('max_object_count', sa.Integer(), nullable=True),
+        sa.Column('default', sa.Boolean(), nullable=False)
+    )
+
+    op.execute("INSERT INTO billing_plan (name, max_request_count, max_object_count, \"default\") VALUES ('Developpeur (3000 requetes max)',3000,3000,true),('Entreprise (requetes illimitees)',NULL,NULL,false);")
```

Comment:
I not agree with that! the enterprise plan should be the default plan, actually all media except the sncf one create user with unlimited calls.